addons/markers/functions/fnc_movePFH.sqf

```diff
@@ -0,0 +1,38 @@
+/*
+ * Author: chris579
+ * When the marker is being moved.
+ *
+ * Arguments:
+ * 0: Marker data <ARRAY>
+ * 1: PFH ID <NUMBER>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [[MARKER], 5] call ace_markers_fnc_movePFH
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+params ["_args", "_idPFH"];
+_args params ["_marker"];
+
+if (isNull (findDisplay 12 displayCtrl 51) || {!(player getVariable [QGVAR(moveInProgress), false])}) exitWith {
+    [_idPFH] call CBA_fnc_removePerFrameHandler;
+
+    private _origin = ACE_player getVariable [QGVAR(movedMarkerOrigin), getMarkerPos _marker];
+
+    if !([QGVAR(markerMoveEnded), [ACE_player, _marker, _origin, getMarkerPos _marker]] call CBA_fnc_localEvent) exitWith {
```

Comment:
Event doesn't exist in ace, is this just so something external can cancel marker movement?
`if (nil) ...` always worries me a little, but it should be fine as is